### ISSUE TITLE
Exceeded maximum allowed number of Trackers

### ISSUE THREAD

**Issue by oishi89** [jeffreybergier/Blog-Getting-Started-with-Vision#3]
Hi there 
After a few time running I had got the error below. Do you have any ideas ? 
Throws: Error Domain=com.apple.vis Code=9 "Internal error: Exceeded maximum allowed number of Trackers for a tracker type: VNObjectTrackerType" UserInfo={NSLocalizedDescription=Internal error: Exceeded maximum allowed number of Trackers for a tracker type: VNObjectTrackerType}

**Comment by jeffreybergier** [comment]
Hi Oishi, I do remember seeing this error. But I don't remember what/if I did anything to fix it. Is it happening every time for you or was it just a one off issue? Does it happen if you download the sample project and run it?

**Comment by oishi89** [comment]
Hi @jeffreybergier 
I got this error every time. Just around 8 - 10 times recreate a new observation. I got the issue in your project as well.

**Comment by jeffreybergier** [comment]
Yeah, sorry, I have no idea what is causing that. A quick google search didn't reveal anything useful. Just double check that your Xcode is up to date and that your iPhone is fully up to date.

**Comment by oishi89** [comment]
I'm using Xcode Version 9.1 beta (9B46) but my Iphone hasn't been 11.1 yet. I'm trying an upgrading now
